packages/holocron-dev-server/src/utils/language-packs.js

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2021 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+import path from 'path';
+import jsonParse from 'json-parse-context';
+
+import { volume, ufs } from './virtual-file-system';
+import { getModulesPath, getLocalesPathForModule } from './paths';
+import { logModuleLanguagePacksLoaded } from './logs';
+import { createLanguagePackWatcher } from './watcher';
+
+export const localePathName = 'locale';
+export const languagePackFileNames = ['copy', 'integration'];
+
+export function readJsonFile(filePath) {
+  const json = ufs.readFileSync(filePath).toString();
+  return jsonParse(json);
+}
+
+export function loadModuleLanguagePack({
+  moduleName, modulePath, localePath = localePathName, locale,
+}) {
+  let languagePackPath = path.join(modulePath, localePath, locale);
+
+  if (!ufs.existsSync(languagePackPath)) {
+    languagePackPath = path.join(modulePath, localePath, `${locale}.json`);
+  }
+
+  if (ufs.existsSync(languagePackPath)) {
+    const fileStats = ufs.statSync(languagePackPath);
+
+    if (fileStats.isDirectory()) {
+      const languagePack = {};
+      const links = {};
+      const scan = [
+        ...languagePackFileNames,
+        locale,
+        moduleName,
+      ];
+      scan
+        .map((fileName) => `${fileName}.json`)
+        .concat('links')
```

Comment:
this shouldn't be hardcoded, `links` is only an example. this could have any name

Reply:
Could you explain more. From what I understood, `links` was an expected directory.

Reply:
May have misunderstood actually. Are you saying any subdirectory is usable and bundled into the language pack?

Reply:
yes they could be anything. locale -> en-us -> names || links || places etc